.circleci/config.yml

```diff
@@ -87,8 +87,16 @@ jobs:
             echo ""
             docker-compose -f docker-compose.yml -f docker-compose-auditor.yml up scalardl-ledger-schema-loader-cassandra
             docker-compose -f docker-compose.yml -f docker-compose-auditor.yml up scalardl-auditor-schema-loader-cassandra
-            docker-compose -f docker-compose.yml -f docker-compose-auditor.yml up -d scalar-ledger scalar-auditor ledger-envoy auditor-envoy
-            docker-compose -f docker-compose.yml -f docker-compose-auditor.yml up -d scalar-ledger-as-client scalar-audior-as-client
+            docker-compose -f docker-compose.yml -f docker-compose-auditor.yml up -d scalar-ledger
+            docker-compose -f docker-compose.yml -f docker-compose-auditor.yml up -d scalar-auditor
+            docker-compose -f docker-compose.yml -f docker-compose-auditor.yml up -d ledger-envoy
+            docker-compose -f docker-compose.yml -f docker-compose-auditor.yml up -d auditor-envoy
+            docker-compose -f docker-compose.yml -f docker-compose-auditor.yml up -d scalar-ledger-as-client
+            docker-compose -f docker-compose.yml -f docker-compose-auditor.yml up -d scalar-audior-as-client
+            sleep 5
+            docker restart scalardl-samples-scalar-ledger-1 # 念のために
+            docker restart scalardl-samples-scalar-auditor-1 # 念のために
```

Comment:
Should be "# just in case" ?? (Web might as well)

Reply:
No problem.
Let me change it.
(Actually maybe the comments are not necessary w)